bin/nexus-enable-rut-auth

```diff
@@ -1,12 +1,12 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-# Copyright 2020 Hewlett Packard Enterprise Development LP
+# Copyright 2022 Hewlett Packard Enterprise Development LP
 
 . "$(dirname "$0")/nexus-ready"
 
 # requires the setup_capability script
 
-echo -n >&2 "Enable Rut Auth..."
+echo -n >&2 "Enable Rut Auth capability..."
```

Comment:
Is this idempotent?  What is the effect of running this multiple times?

Reply:
Running multiple times simply reconfigures the capability (i.e., enables it and sets the HTTP header to `X-WEBAUTH-USER`). You can verify the behavior by running `hack/nexus-setup` repeatedly.

Reply:
I just updated this script to make the `httpHeader` customizable.